### ISSUE TITLE
Convert formulas to LaTeX

### ISSUE THREAD

**Issue by Webysther** [Webysther/e-calculo#2]

**Comment by gpr1** [comment]
Comecei a trabalhar nisso. A ideia é utilizar Mathjax para renderizar isso nos navegadores? Por enquanto tenho deixado o código como comentário nos html.

**Comment by Webysther** [comment]
Exatamente. Por enquanto pode substituir diretamente no HTML, podemos reutilizar sem muito esforço quando migrarmos o HTML para Markdown.
